### ISSUE TITLE
Ability to clear the dateRange

### ISSUE THREAD

**Issue by rocioimpa** [dilipchandima/rn-select-date-range#5]
Hi, I see there is no way to clear the ranges from the calendar once they are set. Is there a way to do this? 
My code is:

``` javascript
const [range, setRange] = useState({});

const handleDateRangeChange = (value) => {
    setRange(value);
};

const handleButtonPress = useCallback(() => {
    setDateRange({});
};

return(
    <DateRangePicker
            onSelectDateRange={handleDateRangeChange}
            blockSingleDateSelection={false}
            responseFormat="DD-MM-YYYY"
            maxDate={moment().add(1, "year")}
            minDate={moment().subtract(2, "year")}
            selectedDateContainerStyle={styles.selectedDateContainerStyle}
            selectedDateStyle={styles.selectedDateStyle}
     />
     <Button onPress={handleButtonPress} mode='contained'>Clear</Button>);
```
Setting the state back to an empty object clears the variable, but the calendar remains with the selected dates marked.

**Comment by drakyone** [comment]
Same problem here

**Comment by dilipchandima** [comment]
We are working on a fix for this issue.

**Comment by dilipchandima** [comment]
@rocioimpa @drakyone I created a clear button for the component, and trying to publish the package, I published the package with public access but it is not reflecting on NPM, I opened a ticket with them I will let you know when the issue is fixed.

**Comment by drakyone** [comment]
> @rocioimpa @drakyone I created a clear button for the component, and trying to publish the package, I published the package with public access but it is not reflecting on NPM, I opened a ticket with them I will let you know when the issue is fixed.

is it possible to make component controlled? (to set default value or something)

**Comment by rocioimpa** [comment]
> @rocioimpa @drakyone I created a clear button for the component, and trying to publish the package, I published the package with public access but it is not reflecting on NPM, I opened a ticket with them I will let you know when the issue is fixed.

Awesome, thanks! Is there a way to control it? Like, if I set the state to the inital state (empty object), the calendar resets itself? Anyways this is great, helps a lot!

**Comment by drakyone** [comment]
> @rocioimpa @drakyone I created a clear button for the component, and trying to publish the package, I published the package with public access but it is not reflecting on NPM, I opened a ticket with them I will let you know when the issue is fixed.

I had the same issue with npm today, just need to wait some time.
 thanks for this beatiful library by the way!

**Comment by dilipchandima** [comment]
yeah seems like they are having an issue - https://status.npmjs.org/

**Comment by rocioimpa** [comment]
Indeed, the library is great, very easy to use 😃

**Comment by dilipchandima** [comment]
The new change is available now, it is Version 3.0.1
